packages/console/src/cloud/pages/Main/Tenants.tsx

```diff
@@ -1,4 +1,4 @@
-import { type TenantInfo, TenantTag } from '@logto/schemas';
+import { type TenantInfo, TenantTag } from '@logto/schemas/models';
```

Comment:
nit: The extra path `/models` is not necessary if you have indexed everything well in the schemas